pylint/checkers/threading_checker.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+from astroid import nodes
+
+from pylint import interfaces
+from pylint.checkers import BaseChecker, utils
+from pylint.checkers.utils import safe_infer
```

Comment:
You are already importing `utils` here. Can we merge these? I think importing `safe_infer` and `check_messages` from `utils` should work well and is what we normally do for these functions in other files.

Reply:
Merged